dali/operators/reader/loader/coco_loader.h

```diff
@@ -200,7 +200,7 @@ class DLL_PUBLIC CocoLoader : public FileLabelLoader {
   void SavePreprocessedAnnotations(const std::string &path, const ImageIdPairs &image_id_pairs);
 
  private:
-  const OpSpec &spec_;
+  const OpSpec spec_;
```

Comment:
That's the only extra copy now. The code would be too brittle or messy without it.